packages/core/src/database/index.ts

```diff
@@ -151,6 +155,11 @@ export const createDatabase = ({
 
     const kyselyDialect = new KyselyPGlite(driver.instance).dialect;
 
+    await driver.instance.query(
+      `CREATE SCHEMA IF NOT EXISTS "${preBuild.namespace}"`,
+    );
+    await driver.instance.query(`SET search_path TO ${preBuild.namespace}`);
```

Comment:
Do we need to escape the namespace here with double quotes? Same question for a few queries below in the role setup.